skyportal/services/test_server/test_api_server.py

```diff
@@ -79,6 +96,41 @@ def lt_request_matcher(r1, r2):
     )
 
 
+def lco_request_matcher(r1, r2):
+    """
+    Helper function to help determine if two requests to the LCO API are equivalent
+    """
+
+    # A request matches an LCO request if the URI and method matches
+
+    r1_uri = r1.uri.replace(":443", "")
+    r2_uri = r2.uri.replace(":443", "")
+    delete_pattern = r"/api/requestgroups/[0-9]+/cancel/"
+    update_pattern = r"/api/requestgroups/[0-9]+/"
+    submit_pattern = r"/api/requestgroups/"
+
+    # Classify r1
```

Comment:
```
submit_patterns = {
    'submit': r"/api/requestgroups/",
    'delete': ...
}

r1_type = None
for (submit_type, pattern) in submit_patterns.items():
    if re.search(pattern, r1_uri):
        r1_type = submit_type
```

And then I'd probably turn that into a function, so you can do:

```
r1_type = submit_type(r1_uri)
r2_type = submit_type(r2_uri)
```

Reply:
@kmshin1397 This is your code. Do you agree / want to push it?

Reply:
Agreed, I just made a PR to your branch for this here: https://github.com/mcoughlin/skyportal/pull/6

Reply:
@kmshin1397 Tests looked good too! Pulled it in.